src/com/parse4cn1/operation/IncrementFieldOperation.java

```diff
@@ -48,7 +48,6 @@ public Object apply(Object oldValue, ParseObject parseObject, String key)
             throws ParseException {
 
         if (oldValue == null) {
-            needIncrement = false;
```

Comment:
needIncrement = true : 
useful when you try to increment without retrieving the object 
`{"seats":{"amount":1,"__op":"Increment"}}`

needIncrement = false:
Cause a problem because it always sends the amount
`{"seats":1}`

So setting it to false gonna miss with the values in the backend